packages/@ama-sdk/core/package.json

```diff
@@ -55,8 +55,10 @@
   "peerDependencies": {
     "@angular-devkit/schematics": "~16.2.0",
     "@angular/common": "~16.2.0",
+    "@angular/cli": "~16.2.0",
```

Comment:
Should they be added in the list of optional, too ?

Reply:
well spotted